packages/@okta/vuepress-site/docs/guides/set-up-self-service-registration/main/index.md

```diff
@@ -0,0 +1,177 @@
+
+When you enable a self-service registration policy, Okta enforces uniqueness for all primary email addresses. When registering, users need to provide their email address. Okta automatically uses the email address for the username and primary email address.
+
+> **Note:** The self-service registration password policy doesn't support "does not contain first name" and "does not contain last name" complexity requirements.
+
+1. In the Admin Console, go to **Directory** and then **Self-Service Registration**.
+
+2. If this is your first time on this page, click **Enable Registration**. Otherwise, click **Edit**.
+
+3. In the **ACCOUNT** section, you have the option to add a sign-up link and to assign the user to an existing group automatically:
+
+    * **Add to Sign-In widget:** (Optional) Select if you want to add a sign-up link to your Okta-hosted sign-in page. Adding a sign-up link to the Widget eliminates the need to configure a link using JavaScript in the Customize Sign-In Page editor.
+
+    * **Assign to group:** (Optional) Enter the existing group name that users are automatically added to when they self register. Group membership determines which password policy is applied during registration. If no group is specified, Okta applies the Default password policy.
+
+4. In the **REGISTRATION FORM** section, you have the option to add a customized label for the Email and Password fields and to define which fields are required. The fields in this section are what the user sees when they register.
+
+    * **Login field form label:** (Optional) Enter a label for the **Email** field on the Create Account registration form.
+
+    * **Password field form label:** (Optional) Enter a label for the **Password** field on the Create Account registration form.
+
+    * **Registration form fields:** (Optional) The `firstName` and `lastName` fields are required and can't be removed. You can accept the default **Form label** value or optionally enter a different field name.
+
+        * To include more fields: click **Add Field** and then select an existing profile attribute from the list.
+
+          > **Note:** If you want to add new profile attributes, click **Configure the Okta user profile** to add them to the Okta user profile. They are then available in the list on this page. Be sure to set the **User permission** drop-down box to **Read - Write** if you want your users to be able to modify the new attributes after registration.
+
+        * To make a field mandatory: select the **Required** check box.
+        * To change the order of the fields on the form: grab the dotted vertical bar on the left to drag a field to a new location.
+        * To remove a field: click **X**.
+
+    The registration form supports the following data types:
+    * string
```

Comment:
Initial uppercase each bullet item.

Reply:
Good call. I've fixed these in my commit. Thanks!